README.md

```diff
@@ -140,4 +142,21 @@ Note: that if you have elements that are not numbers it will concatenate string.
     ``` array.sum(x => x * 2)```
 - **Parameters**
     * property
-        Function or null value
\ No newline at end of file
+        Function or null value
+
+## 13. max(comparator) method
+Return the maximum value on the collection, a null value is return if
+the array is empty. If not `comparator` is specified then it evaluates
+the array elements as if they are numbers.
+
+- **Syntax**
+    ```
+        array.max(function(a,b) { 
```

Comment:
this example is wrong, since it's not an *arrow function* it has to have a `return` statement, use `array.max((a, b) => a.length - b.length)` instead.